src/api/spec/controllers/status/reports_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Status::ReportsController, type: :controller do
+  let(:user) { create(:confirmed_user) }
+  let(:project) { create(:project_with_repository) }
+  let(:source_project) { create(:project_with_package) }
+  let(:target_project) { create(:project_with_package) }
+  let(:bs_request) do
+    create(:bs_request_with_submit_action,
+           creator: user,
+           source_project: source_project,
+           source_package: source_project.packages.first,
+           target_project: target_project,
+           target_package: target_project.packages.first)
```

Comment:
`target_project.packages.first` is repeated, maybe assign it to a variable? :thinking:

Reply:
No, ``target_project.packages.first`` vs ``source_project.packages.first``